zerver/views/home.py

```diff
@@ -299,6 +299,18 @@ def home_real(request: HttpRequest) -> HttpResponse:
 def desktop_home(request: HttpRequest) -> HttpResponse:
     return HttpResponseRedirect(reverse('zerver.views.home.home'))
 
+def get_isolated_page(request: HttpRequest) -> bool:
+    '''Accept a GET param `?nav=no` to render an isolated, navless page.'''
+    return request.GET.get('nav') == 'no'
+
+def terms_view(request: HttpRequest) -> HttpResponse:
+    return render(request, 'zerver/terms.html',
+                  context={'isolated_page': get_isolated_page(request)})
+
+def privacy_view(request: HttpRequest) -> HttpResponse:
+    return render(request, 'zerver/privacy.html',
+                  context={'isolated_page': get_isolated_page(request)})
+
```

Comment:
These should go in a new `zerver/views/portico.py`, I think.  `home.py` is primarily for the webapp itself (probably wants to be renamed), and I don't want to expand it being cluttered with other stuff.

(We also probably want to rename it).  I'll extend this series to address that.